Lib/ufo2ft/featureWriters/baseFeatureWriter.py

```diff
@@ -158,11 +158,13 @@ def _insert(
 
         statements = feaFile.statements
 
-        # Collect insert markers in blocks
-        insertComments = self.context.insertComments
+        for feature in features:
+            feature._inserted = False
```

Comment:
here you're relying on the fact that ast.Feature object has a `__dict__` and is mutable, which may not necessarily always be the case and is arguably an implementation detail? maybe you could make a dict keyed by `id(feature)` (supposedly unique to the object) and value being a bool (inserted or not)?